src/fonts/bravura_glyphs.js

```diff
@@ -1,1055 +1,1151 @@
 export const BravuraFont = {
-  'glyphs': {
```

Comment:
You mentioned in the bug that you used a different Bravura OTF file to generate these, can you please add that OTF file to this repo please?